project/Protobuf.scala

```diff
@@ -32,7 +32,7 @@ object Protobuf {
     Compile / unmanagedJars ++= Seq(
         baseDirectory.value / ".." / "akka-protobuf-v3" / "target" / s"akka-protobuf-v3-assembly-${version.value}.jar"),
     protoc := "protoc",
-    protocVersion := "3.9.0",
+    protocVersion := "3.9.2",
```

Comment:
do have to keep this in sync with the `protobuf-java` dependency? can we define it at one place?

Reply:
Not for minor versions but let's do that anyway

Reply:
will you change to a shared `val` or do we leave it as this?